ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -160,6 +211,29 @@ message GetUserResponse {
     User user = 1;
 }
 
+// Required authorization: ``HasRight(ParticipantAdmin)``
+// NOTE: The authenticated participant user is not allowed to deactivate itself.
```

Comment:
```suggestion
```
Users don't need to plan for this, as it is somewhat obvious; and otherwise, the error will do a good job at explaining the restriction.

Reply:
Ok